resources/articles/oauth/jwks.md

```diff
@@ -0,0 +1,58 @@
+:title GET /oauth/jwks
+
+:aside
+
+## See also
+
+* [POST /oauth/token](/oauth/token/)
+* [POST /oauth/ro](/oauth/ro/)
+* [POST /oauth/introspect](/oauth/introspect/)
+* [GET /oauth/userinfo](/oauth/userinfo/)
+* [GET /oauth/authorize](/oauth/authorize/)
+
+:body
+
+SPiD publish the public RSA keys as a JSON Web Key (JWK) set. These web keys are
```

Comment:
Replace paragraph with: "SPiD publishes the public part of the RSA keys used to sign tokens on the `/oauth/jwks` endpoint. This enables applications to verify the validity of tokens without making a request to [/oauth/introspect](/oauth/introspect/)."